<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editshortcutwidget</class>
 <widget class="QWidget" name="editshortcutwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shortcut:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QRadioButton" name="builtindbusRbt">
       <property name="text">
        <string>Built in Dbus</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okbt">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-ok-apply"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelbt">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-cancel"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="descriptionTbox"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QRadioButton" name="commandRbt">
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Command:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="commandTbox"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0">
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Bus:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="busCbox">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Session</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>System</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Path:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="pathTbox"/>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="interfaceTbox"/>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="methodTbox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Method:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Service:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Interface:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="serviceTbox"/>
           </item>
           <item row="4" column="2">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="builtindbusLwidget"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="customdbusRbt">
       <property name="text">
        <string>Custom Dbus</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QPushButton" name="shortcutbt">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>shortcutbt</tabstop>
  <tabstop>descriptionTbox</tabstop>
  <tabstop>commandRbt</tabstop>
  <tabstop>customdbusRbt</tabstop>
  <tabstop>builtindbusRbt</tabstop>
  <tabstop>commandTbox</tabstop>
  <tabstop>okbt</tabstop>
  <tabstop>cancelbt</tabstop>
  <tabstop>serviceTbox</tabstop>
  <tabstop>pathTbox</tabstop>
  <tabstop>interfaceTbox</tabstop>
  <tabstop>methodTbox</tabstop>
  <tabstop>busCbox</tabstop>
  <tabstop>builtindbusLwidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
